api/instance/v1/image_utils.go

```diff
@@ -0,0 +1,50 @@
+package instance
+
+import (
+	"time"
+
+	"github.com/scaleway/scaleway-sdk-go/internal/async"
+	"github.com/scaleway/scaleway-sdk-go/internal/errors"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+// WaitForImageRequest is used by WaitForImage method.
+type WaitForImageRequest struct {
+	ImageID string
+	Zone    scw.Zone
+	Timeout time.Duration
```

Comment:
I know instance doesn't use pointers here, but shouldn't it be better?

Reply:
I want to stay consistent with `WaitForServer`